examples/plot_fmri_data_example.py

```diff
@@ -152,18 +153,23 @@ def preprocess_haxby(subject=2, memory=None):
 SVR_permutation_test_inference = False
 if SVR_permutation_test_inference:
     # We computed the regularization parameter by CV (C = 0.1)
-    pval_corr_svr_perm_test, one_minus_pval_corr_svr_perm_test = permutation_test_cv(
-        X, y, n_permutations=50, C=0.1
+    estimator = LinearSVR(C=0.1)
```

Comment:
C is the regularization parameter, it is not optimized via CV here. Or am I missing something? 
I suggest using something like randomized search. 
```suggestion
    estimator = RandomizedSearchCV( 
        LinearSVR(random_state=42), 
        param_distributions={ "C": np.logspace(-3, 3, 10), }, 
        n_iter=10, 
        n_jobs=5, 
        random_state=42, 
    )
```

Reply:
I didn't provide an optimisation by CV because, in the original example, it wasn't the case.
Nevertheless, we need to be careful with optimisation because it will increase the time for running examples.

Reply:
Do you have an estimate of the compute time ?

Reply:
On my computer, the example without the CV is: 5m39s and with CV is: 7m16s.
In estimation, it increases 2 minutes the time of calculation. 

One solution can be to store the best value of the parameter and avoid doing the refitting each time. .

Reply:
Also, there's L153 `SVR_permutation_test_inference = False` so is this even running in the CI? 
If not, I suggest leaving the CV to show this possibility to the user without increasing CI time.

Reply:
I put SVR_permutation_test_inference = True for my test. 

Including, SVR_permutation_test_inference options in my computation time, I get:
SVR_permutation_test_inference = True and CV: 7m16s.
SVR_permutation_test_inference = True and No CV: 5m39s
SVR_permutation_test_inference = False and CV: 1m57s.
SVR_permutation_test_inference = False and No CV: 1m48s.

Reply:
@bthirion

Reply:
For the time being, we should not change this if there is no explicit reason for that (e.g. significant reduction of documentation generation time).

Reply:
> Please use a RidgeCV, so that there is no magic alpha parameter hanging around.
@bthirion Do we do the cross-validation for Ridge and linearSVR or only for Ridge?

Reply:
@bthirion